workloads/virtual-machines/virtualized-hardware-configuration.md

```diff
@@ -90,6 +92,38 @@ spec:
         claimName: myclaim
 ```
 
+### Model
+
+Setting the CPU model is possible via `spec.domain.cpu.model`. The following VM will have a CPU with the `Conroe` model:
+
+```yaml
+metadata:
+  name: myvmi
+spec:
+  domain:
+    cpu:
+      # this sets the CPU model
+      model: Conroe
+    resources:
+      requests:
+        memory: 512M
+    devices:
+      disks:
+      - name: myimage
+        volumeName: myimage
+        disk: {}
+  volumes:
+    - name: myimage
+      persistentVolumeClaim:
+        claimName: myclaim
+```
+
+You can check list of available models [here](https://github.com/libvirt/libvirt/blob/master/src/cpu/cpu_map.xml).
+ 
+**Note**: Be sure that node CPU model where you run a VM, has the same or higher CPU family.
```

Comment:
Please move the note to the beginnig of the section

Reply:
done